src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+function InputWithLabel({ todoTitle, onChange, children }) {
+    const handleTitleChange= onChange;
+
+    const inputRef = React.useRef();
+
+    React.useEffect(() => {
+        inputRef.current.focus();
+    }, []);
+    return (
+        <>
+            <label htmlFor="todoTitle">{children}</label>
+                <input 
+                    type="text" 
+                    id="todoTitle" 
+                    name="title" 
+                    value={todoTitle}
+                    ref={inputRef}
+                    onChange={handleTitleChange}
+                    />
+        </>
```

Comment:
Be careful with your indentation, it might seem like a small thing, but in larger files it really helps to have everything exactly where it should be.

Above, there's no reason for `<input` on line 14 to be indented further than `<label>`.
Also, the `<input>`'s closing tag on line 21 should be in line with it's opening tag on line 14.